packages/destination-actions/src/destinations/talon-one/trackEvent/generated-types.ts

```diff
@@ -2,21 +2,21 @@
 
 export interface Payload {
   /**
-   * Unique identifier of the customer profile associated to the event.
+   * Unique identifier of the customer profile.
    */
-  customer_profile_id: string
+  customerProfileId: string
   /**
-   * It's just the name of your event.
+   * It is just the name of your event.
    */
-  event_type: string
+  eventType: string
   /**
    * Type of event. Can be only `string`, `time`, `number`, `boolean`, `location`
    */
   type: string
   /**
-   * Arbitrary additional JSON data associated with the event
+   * List the AVPs you need to update in the customer profile.
```

Comment:
These attributes are associated with the event and not with the customer profile. Please return it to the way it was

Reply:
Also, I took this description `Arbitrary additional JSON data associated with the event` from [docs.talon.one](https://docs.talon.one/integration-api/#operation/trackEvent)